build.sbt

```diff
@@ -18,8 +20,11 @@ licenses := Seq("Apache V2" -> url("http://www.apache.org/licenses/LICENSE-2.0.h
 resolvers += Resolver.bintrayRepo("guardian", "ophan")
 
 libraryDependencies ++= Seq(
-  "com.gu" %% "acquisition-event-producer" % "2.0.1",
-  "com.gu" %% "support-internationalisation" % "0.6" % "provided"
+  "com.gu" %% "support-internationalisation" % "0.9" % "provided",
+  scalaVersion {
+    case "2.11.8" => "com.gu" %% "acquisition-event-producer" % "2.0.1"
+    case "2.12.4" => "com.gu" %% "acquisition-event-producer-play26" % "4.0.0"
```

Comment:
How about matching prefixes so that this line remains stable with new patch versions of Scala?
```scala
case v if v.startsWith("2.11.") =>
```